device/coordinate_manager.cpp

```diff
@@ -461,7 +461,9 @@ std::shared_ptr<CoordinateManager> CoordinateManager::create_coordinate_manager(
     tt::ARCH arch,
     const size_t tensix_harvesting_mask,
     const size_t dram_harvesting_mask,
-    const size_t eth_harvesting_mask) {
+    const size_t eth_harvesting_mask,
+    const BoardType board_type,
+    bool is_chip_remote) {
```

Comment:
Related to the other comment. Consider this function accepting BlackholeChipType rather then board type and this flag.

Reply:
https://github.com/tenstorrent/tt-umd/pull/437#discussion_r1897984561